net-mail/automx2/automx2-2024.1.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{11,12} )
+
+inherit distutils-r1
+
+DESCRIPTION="Email client autoconfiguration service"
+HOMEPAGE="https://rseichter.github.io/automx2/"
+SRC_URI="https://github.com/rseichter/automx2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="acct-user/automx2
+	dev-python/flask[${PYTHON_USEDEP}]
+	dev-python/flask-migrate[${PYTHON_USEDEP}]
+	dev-python/flask-sqlalchemy[${PYTHON_USEDEP}]
+	dev-python/ldap3[${PYTHON_USEDEP}]
+"
+
+distutils_enable_tests unittest
+
+python_prepare_all() {
+		sed -i -E -e 's/^(packages =).+/\1 automx2/' setup.cfg || die
+		distutils-r1_python_prepare_all
+}
+
```

Comment:
```diff
diff --git a/net-mail/automx2/automx2-2022.1-r1.ebuild b/net-mail/automx2/automx2-2024.1.ebuild
similarity index 90%
copy from net-mail/automx2/automx2-2022.1-r1.ebuild
copy to net-mail/automx2/automx2-2024.1.ebuild
index ead2c323a51f..0bcb4b570a95 100644
--- a/net-mail/automx2/automx2-2022.1-r1.ebuild
+++ b/net-mail/automx2/automx2-2024.1.ebuild
@@ -14,7 +14,7 @@ SRC_URI="https://github.com/rseichter/automx2/archive/${PV}.tar.gz -> ${P}.tar.g
 
 LICENSE="GPL-3+"
 SLOT="0"
-KEYWORDS="amd64"
+KEYWORDS="~amd64"
 
 RDEPEND="acct-user/automx2
 	dev-python/flask[${PYTHON_USEDEP}]
@@ -26,8 +26,8 @@ RDEPEND="acct-user/automx2
 distutils_enable_tests unittest
 
 python_prepare_all() {
-	sed -i -e "/('scripts'/d" setup.py || die
-	distutils-r1_python_prepare_all
+		sed -i -E -e 's/^(packages =).+/\1 automx2/' setup.cfg || die
+		distutils-r1_python_prepare_all
 }
 
 python_test() {
```

Mysterious. Please explain and rationalize.

Reply:
1. The source archive of automx2 2024.1 does not contain `setup.py` anymore.
2. Overriding `package = ...` is required to avoid the `tests` folder being auto-detected for installation (which QA doesn't permit).

Reply:
I would like you to take a closer look at the actual change, and deduce why I would suggest it's unambiguously problematic for reasons that have nothing to do with the code itself.

Assume that I'm aware upstream files sometimes change.